auto-voice-channels.py

```diff
@@ -20,13 +21,20 @@
 from functions import log, echo
 from discord.ext.tasks import loop
 
-logging.basicConfig(level=logging.INFO)
+#logging.basicConfig(level=logging.INFO)
 ADMIN_CHANNEL = None
 ADMIN = None
-
+PRODUCTION_BUILD = True    # IMPORTANT! ---> SET THIS TO FALSE IF NOT CLUSTERING

```

Comment:
Would be better to have this set in config.json (defaulted to False), people self-hosting don't want to edit code just to get set up.

`PRODUCTION_BUILD = cfg.CONFIG['production_build'] if 'production_build' in cfg.CONFIG else False`